src/overlays/actors/ovl_En_Niw_Girl/z_en_niw_girl.h

```diff
@@ -3,12 +3,36 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include "../ovl_En_Niw/z_en_niw.h"
 
 struct EnNiwGirl;
 
+typedef void (*EnNiwGirlActionFunc)(struct EnNiwGirl*, GlobalContext*);
+
 typedef struct EnNiwGirl {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1B0];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[17];
+    /* 0x01F6 */ Vec3s transitionDrawTable[17];
+    /* 0x025C */ EnNiwGirlActionFunc actionFunc;
+    /* 0x0260 */ Vec3s unk_260;
+    /* 0x0266 */ Vec3s unk_266;
+    /*
+    *Controls how many frames she jumps for and how long until she jumps again
```

Comment:
Comments about struct fields should probably go after the field on the same line instead of inserting extra lines into the middle like this